app/Models/Task.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    use HasFactory;
+
+    protected $guarded = [];
+
+    protected $casts = [
+        'closed_at' => 'datetime',
+    ];
+
+    public function recipient()
+    {
+        return $this->belongsTo(User::class, 'recipient_user_id');
+    }
+
+    public function sender()
```

Comment:
suggestion: rename re: migration comment
```suggestion
    /**
     * The user or entity which created the task.
     */
    public function creator()
```